src/ngResource/resource.js

```diff
@@ -294,46 +301,54 @@ function shallowClearAndCopy(src, dst) {
  *
  * ### Credit card resource
```

Comment:
It would be nice if this example and the next one had names that tell what they are showing, e.g. for this one "Basic Usage" and for the second one maybe "Accessing the http response / Promises and Callbacks"

Reply:
+1